charts/kubewarden-controller/Chart.yaml

```diff
@@ -21,9 +21,9 @@ maintainers:
 # This is the chart version. This version number should be incremented each time you make changes
```

Comment:
(review of the PR opening automation)

:heavy_check_mark: 

This PR comes from this successful workflow:
https://github.com/kubewarden/helm-charts/actions/runs/6471156725/job/17673504591

This successful workflow has consumed correct 1.8.0-rc2 images, and the following CRDs:
- controller crd asset ID: 130084126
  matches with the step in the release of the controller that triggered the worklow dispatch:
  https://github.com/kubewarden/kubewarden-controller/actions/runs/6483590687/job/17606538760#step:10:6
- audit-scanner crd asset ID: 130277854
   matches with the step in the release of the audit-scanner that triggered the worklow dispatch:
   https://github.com/kubewarden/audit-scanner/actions/runs/6497505392/job/17647831934#step:11:6